.github/workflows/docker-release.yml

```diff
@@ -64,6 +71,10 @@ jobs:
             jellyfin/jellyfin-vue:stable-ssr
             jellyfin/jellyfin-vue:stable-ssr.${{ github.event.release.tag_name }}
             jellyfin/jellyfin-vue:latest
+            ghcr.io/jellyfin/jellyfin-vue:stable
```

Comment:
Does this push these labels to Docker Hub too?

Reply:
Yup, check the lines above ;)

Reply:
I think @ThibaultNocchi misinterpreted your question.

No, the ones starting with ghcr.io goes to ghcr and the ones that don't have anything use the default repository (DockerHub). [See here](https://docs.docker.com/engine/reference/commandline/tag/)

So, basically, by loading both credentials and tagging with the correct url, each tag goes to the corresponding place.

I tested this personally in this repo: https://github.com/ferferga/selfhosted-mail-forwarding

Reply:
Oh yeah that's how docker pull / pushworks basically so I misinterpreted :p